workspace_status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# p_() takes 2 inputs, the name of a variable and a value. It then prints the
+# variable and the value to be fed into bazel's --workspace_status flag. The
+# interesting thing is that it will *only* print the given value if there is
+# *not* an environment variable with the same name that is already set (with
+# some nonempty string). This way, bazel will either read in these ENV vars, but
+# if they don't exist, default to the value provided to p_().
+p_() {
+    if (( $# == 2 )); then
+        echo "$1" "${!1:-$2}"
+    else
+        return 1
+    fi
+}
+
+git_branch="$(git rev-parse --abbrev-ref HEAD)"
+git_desc="$(git describe --always)"
+img_name="cip"
+
+p_ IMG_REGISTRY gcr.io
```

Comment:
you probably need to prefix most/all of these with `STABLE_`, so that they trigger rebuilds of stamped targets.

otherwise, changes to the values of these won't trigger rebuilds, which can be confusing.

Reply:
Gotcha, will do.